localstack/services/s3/provider.py

```diff
@@ -189,6 +185,13 @@
 from localstack.utils.time import parse_timestamp
 from localstack.utils.urls import localstack_host
 
+if config.LEGACY_V2_S3_PROVIDER:
+    from localstack.services.s3.utils_moto import (
+        get_bucket_from_moto,
+        get_key_from_moto_bucket,
+        is_moto_key_expired,
+    )
+
```

Comment:
this might not be needed, as it would only be loaded if `config.LEGACY_V2_S3_PROVIDER` is True.